tools/genesis/workbook/load_data.go

```diff
@@ -41,7 +42,8 @@ func ParseAllocations(xls *excelize.File) []*AllocationRow {
 		}
 
 		if err = row.FromRow(i, urow); err != nil {
-			log.Panic("could not parse row: ", i+1, err)
+			fmt.Printf("could not parse row: %d (%v)\n", i+1, err)
+			continue
```

Comment:
I thought that Alex asked tool to not produce genesis if there is something off with xls ?

Reply:
The genesis xsl is too broken to not do this....